app.config.ts

```diff
@@ -57,7 +57,13 @@ function docsEntries() {
 export default defineConfig({
 	middleware: "src/middleware/index.ts",
 	server: {
-		preset: "netlify-edge",
+		preset: "netlify",
+		prerender: {
+			crawlLinks: true,
+			autoSubfolderIndex: false,
+			failOnError: true,
+			ignore: [/\{\getPath}/],
+		},
```

Comment:
@nksaraf sorry it took me a while to get here, but I think crawlLinks can be enabled again. Would you mind checking if there's any other settings you'd recommend?